src/SecureRoute.tsx

```diff
@@ -21,7 +25,8 @@ const SecureRoute: React.FC<{
   ...routeProps 
 }) => { 
   const { oktaAuth, authState, _onAuthRequired } = useOktaAuth();
-  const match = useRouteMatch(routeProps);
+  const { path, caseSensitive } = routeProps;
+  const match = path ? useMatch.call(null, { path, caseSensitive }) : null;
```

Comment:
`useMatch` doesn't seem to be matching a route when `basename` property is set
we could provide `useLocation().pathname` value(which includes basename) as `path` parameter for `useMatch()`